### DISCUSSION TITLE
Resolve IP addresses in dashboard with DNS

### DISCUSSION THREAD

**Discussion by ponte19** [Security-Onion-Solutions/securityonion#11801] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

128

### Storage for /

1TB

### Storage for /nsm

1TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Is it possible to resolve the IP addresses in the dashboard with a DNS server in the network? Currently I can see all the IP addresses but it is not very convenient. Hostnames would be easier.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by robbiemarshall** [comment, marked as answer]
You can enable DNS reverse lookup by going in to SOC and then Administration > Configuration > SOC >Config > Server > Client > enableReverseLookup and then set the current grid value to "true"
